output/plugins/layout/xml/layout.xml

```diff
@@ -76,12 +76,20 @@ Written by
       <option name="wxHORIZONTAL" 	help="Align items horizontally"/>wxVERTICAL</property>
   </objectinfo>
 
-  <objectinfo class="wxWrapSizer" icon="sizer.xpm" type="sizer">
+  <objectinfo class="wxWrapSizer" icon="wrap_sizer.xpm" type="sizer">
 	<inherits class="sizer" />
     <property name="name" type="text" hidden="1">wSizer</property>
     <property name="orient" type="option">
       <option name="wxVERTICAL" 	help="Align items vertically"/>
-      <option name="wxHORIZONTAL" 	help="Align items horizontally"/>wxHORIZONTAL</property>
+      <option name="wxHORIZONTAL" 	help="Align items horizontally"/>
+      wxVERTICAL
```

Comment:
If I understand you correctly [here](https://github.com/wxFormBuilder/wxFormBuilder/issues/211#issuecomment-402530838), it would be better to change this to `wxHORIZONTAL`, right?

Reply:
Yes, because that is the default value of this sizer